docs/core/logger.md

```diff
@@ -319,6 +319,22 @@ Logger can optionally log uncaught exceptions by setting `log_uncaught_exception
     --8<-- "examples/logger/src/logging_uncaught_exceptions_output.json"
     ```
 
+#### Stack trace logging
+
+By default, the Logger will automatically log the full stack trace in JSON format when using `logger.exception`. If you want to disable this feature, set `serialize_stacktrace=False` during initialization."
```

Comment:
```suggestion
By default, the Logger will automatically includes the full stack trace in JSON format when using `logger.exception`. If you want to disable this feature, set `serialize_stacktrace=False` during initialization."
```

Question: If the new behaviour is True, will this not be a breaking change, breaking the expected format that people are used to?

Reply:
We are adding a new field called `stack_trace` and preserving the current fields `exception` and `exception_name` using the same format. I don't see a scenario where it could break a customer, perhaps a client making data validation when exporting logs? I see great added value in logging the stack trace as JSON and allowing the customer to analyze it OR easily identify errors.

In my opinion, we can apply the principle of “better to ask for forgiveness than permission” here.

WDTY?

Reply:
You are absolutely right, I thought for a moment that we were changing an existing field.

Reply:
@leandrodamascena **include**, not **includes**